week2/materials/homework/assignments.md

```diff
@@ -4,51 +4,63 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `https://s3.console.aws.amazon.com/s3/buckets/svetlana-week2?region=us-east-1&tab=objects`
+
+url: `http://svetlana-week2.s3-website-us-east-1.amazonaws.com/`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
+service name with explanation: `Amazon CloudFront is a web service that speeds up distribution of your static and dynamic web content, such as .html, .css, .js, and image files, to your users. CloudFront delivers your content through a worldwide network of data centers called edge locations. When a user requests content that you're serving with CloudFront, the request is routed to the edge location that provides the lowest latency (time delay), so that content is delivered with the best possible performance. You can configure CloudFront to require that viewers use HTTPS so that connections are encrypted when CloudFront communicates with viewers. You also can configure CloudFront to use HTTPS with your origin so that connections are encrypted when CloudFront communicates with your origin. CloudFront assigns a default domain name to your distribution, for example, d111111abcdef8.cloudfront.net. If you use this domain name, then you can use the CloudFront default SSL/TLS certificate already selected for your distribution. If you use a different domain name for your distribution, then it's a best practice to do one of the following to avoid domain-name-related certificate warnings: request a public certificate from AWS Certificate Manager or import certificates into AWS Certificate Manager. AWS Certificate Manager is a service that lets you easily provision, manage, and deploy public and private SSL/TLS certificates for use with AWS services and your internal connected resources. With AWS Certificate Manager, you can quickly request a certificate, deploy it on ACM-integrated AWS resources, such as Elastic Load Balancers, Amazon CloudFront distributions, and APIs on API Gateway, and let AWS Certificate Manager handle certificate renewals. `
+
+url: `https://aws.amazon.com/premiumsupport/knowledge-center/install-ssl-cloudfront/`
 
 **Assignment 6:**
 
 Figure out and write down the price per month of storing 51TB on S3.
 
-Price: `?`
+Price: `1.172$`
```

Comment:
The answer is correct and I know it is not specified but would be nice to write maybe one line how you calculated it :) 
example:
first 50TB is $0,023 pr. GB
Next 450TB is $0,022 pr. GB
1 TB = 1000GB
therefore:
50000 * 0,023+1000 * 0,022 = $1.172 pr. month